apps/web/src/pages/auth/components/HubspotSignupForm.tsx

```diff
@@ -0,0 +1,134 @@
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useMutation } from '@tanstack/react-query';
+import decode from 'jwt-decode';
+
+import { JobTitleEnum } from '@novu/shared';
+import type { ProductUseCases, IResponseError, ICreateOrganizationDto, IJwtPayload } from '@novu/shared';
+
+import { api } from '../../../api/api.client';
+import { useAuthContext } from '../../../components/providers/AuthProvider';
+import { useVercelIntegration, useVercelParams } from '../../../hooks';
+import { ROUTES } from '../../../constants/routes.enum';
+import { HubspotForm } from '@novu/shared-web';
+import { HUBSPOT_FORM_IDS } from '../../../constants/hubspotForms';
+import SetupLoader from './SetupLoader';
+
+export function HubspotSignupForm() {
```

Comment:
How does this form function when a User is developing locally and/or has no internet connection? Will the onboarding flow be blocked?

We may need to consider adding a callback in the HubspotForm component to detect when the form is not downloaded, to handle the fallback case.

Reply:
For local machine, I have added env check on parent page.

Could you help me with fallback in case of no internet?

Reply:
The approach we have added above to only render this form when `NODE_ENV=production` will cater to the case we are describing above, ensuring that local development environment always receives the non-Hubspot form.